src/Arbitrable.ts

```diff
@@ -2,7 +2,7 @@ import { Evidence } from '../types/schema'
 import { EvidenceSubmitted } from '../types/templates/Arbitrable/Arbitrable'
 
 export function handleEvidenceSubmitted(event: EvidenceSubmitted): void {
-  let id = event.transaction.hash.toHex() + event.logIndex.toHex()
+  let id = event.transaction.hash.toHex() + event.logIndex.toHex().slice(2).padStart(4, '0')
```

Comment:
I was wondering about this; would it make sense to add a separator like `-` or `:` so it's easier for frontends to split this into the tx hash and log index if they have to?

Reply:
What's the reason for this fix? Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Reply:
What's the reason for this fix? Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Reply:
Well, the way it is now, it should be easy with slice, as both parts are fixed length.